### ISSUE TITLE
The writings are difficult to read

### ISSUE THREAD

**Issue by Alexboy2334** [Hydr8gon/sm64#9]
The super mario 64 font is quite peculiar and reduced on a small screen with few pixels is quite difficult to read. How hard would it be to change the font?

**Comment by Hydr8gon** [comment]
Indeed, the font is a bit hard to read given the DSi's resolution. My original vision was to have SM64 on the DSi as true to the original as possible, including the somewhat illegible font, heh. There were plans for a "non-vanilla" branch meant for changes that optimize the experience for the DSi, but I haven't really done anything for it yet. My interest mostly lies in the porting process, so stuff like enhancements tend to fall lower on my priority list.
